src/tye/Program.DefaultOptionsMiddleware.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.Linq;
+
+namespace Microsoft.Tye
+{
+    public static partial class Program
+    {
+        private const string DefaultOptionsEnvVarPrefix = "TYE_";
+        private const string DefaultOptionsEnvVarPostfix = "_ARGS";
+
+        private static void DefaultOptionsMiddleware(InvocationContext context)
```

Comment:
Is it feasible to add a test for this middleware? Is there a mock invocation context you could have to parse?

Reply:
I think that i can test it. Will add some tests.